### DISCUSSION TITLE
GitHub CoPilot not installed vscode spaces

### DISCUSSION THREAD

**Discussion by AhmedSyeda** [skills/.github#309] [Q&A]
Hello Community.
After completing all the steps from the readme file to start the course I successfully created the vs codespaces.  I'm not able to see GitHub CoPilot. Upon futher exploration found this message. GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as AhmedSyeda." I checked the access and I'm logged in as Ahmed syeda on github as well. I don't understand what is missing from an authentication point of view?
here is the output log-
[INFO] [auth] [2023-07-23T22:56:30.835Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-07-23T22:56:30.838Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as AhmedSyeda."

**Comment by cmwilson21** [comment]
@AhmedSyeda Thanks for opening a discussion but if you are still running into this issue, reach out to our [support team](https://support.github.com/request). 💛
